README.md

```diff
@@ -97,11 +97,10 @@ header('Content-Type: application/json');
  
 // Try to do the POST with orderRequest Model.
 try {
-    $postResponse = $divideIq->request('stockbase_orderrequest', $orderRequest->toArray(), 'POST');
+    $postResponse = $divideIq->request('stockbase_orderrequest', $orderRequest->jsonSerialize(), 'POST');
```

Comment:
The only thing you need to do is change the Docbloc to say that `$payload` can also be an object. And fix usage of the `$payload` in the HTTP GET scenario.